src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -1426,6 +1426,9 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="OM_CannotRemoveMetadataOriginatingFromItemDefinition">
     <value>Cannot remove the metadata named "{0}" as it originates from an item definition, rather than being directly defined on this item.</value>
   </data>
+  <data name="OM_CannotSplitItemElementWhenSplittingIsDisabled">
+    <value>The requested operation needs to split the item element at location {0} into individual components but item element splitting is disabled with {1}.</value>
```

Comment:
I know this is the phrasing I suggested but I have a better idea. 😇 

Instead of "into individual components", "into individual elements"--then it's more precisely describing what would happen to the XML.